src/components/Select/GroupHeading.tsx

```diff
@@ -28,11 +28,13 @@ export type GroupHeadingClasses = GetClasses<typeof useStyles>;
 
 export interface GroupHeadingProps {
   className?: string;
+  ['data-select-role']: 'heading';
 }
 
 export const GroupHeading: React.FC<GroupHeadingProps> = ({
   children,
   className,
+  ['data-select-role']: dataSelectRole,
```

Comment:
I did this just to keep the select role from being forwarded to the `li` element.

Reply:
Works for me 👍